freeipa/src/main/java/com/sequenceiq/freeipa/entity/UserSyncStatus.java

```diff
@@ -32,6 +32,12 @@
 
     private Long lastFullSyncEndTime;
 
+    @OneToOne
+    private Operation lastRequestedFullSync;
```

Comment:
so this does not get updated when a request is rejected because there is an ongoing user sync right?